get_trust_anchor.py

```diff
@@ -385,12 +385,15 @@ def main():
     trust_anchor_filename = "root-anchors.xml"
     signature_filename = "root-anchors.p7s"
     icann_ca_filename = "icanncacert.pem"
+    temp_files = [trust_anchor_filename, signature_filename, icann_ca_filename]
```

Comment:
We could do that, but it would be more complicated. Doing it this way also makes things easier to debug if the program dies in the middle.